lib/periodic_records/model.rb

```diff
@@ -76,6 +76,10 @@ def periodic_dup
       dup
     end
 
+    def periodic_increment
```

Comment:
maybe we can call this `periodic_precision`? wdyt?

Reply:
Perhaps `splitting_step` or `record_split_step`, because it's only used on overlapping record splitting?